django/urls/resolvers.py

```diff
@@ -206,6 +207,9 @@ def _route_to_regex(route, is_endpoint=False):
     For example, 'foo/<int:pk>' returns '^foo\\/(?P<pk>[0-9]+)'
     and {'pk': <django.urls.converters.IntConverter>}.
     """
+    if any((s in route) for s in string.whitespace):
+        raise ImproperlyConfigured("URL route '%s' contain invalid character space." % route)
```

Comment:
```suggestion
        raise ImproperlyConfigured("URL route '%s' contains invalid whitespace." % route)
```

Reply:
Maybe without `invalid`? `"URL route '%s' contains whitespace."` or `"URL route '%s' cannot contain whitespace."`

Reply:
Let's go with `...cannot contain whitespace.` (I briefly considered that, so it coming up twice, we'll take as a message from the universe.)